mkosi/__init__.py

```diff
@@ -568,8 +571,23 @@ def script_maybe_chroot_sandbox(
 ) -> Iterator[list[PathString]]:
     options = ["--dir", "/work/src", "--chdir", "/work/src", *options]
 
+    suppress_chown = parse_boolean(context.config.environment.get("MKOSI_CHROOT_SUPPRESS_CHOWN", "0"))
+    if suppress_chown:
+        options += ["--suppress-chown"]
```

Comment:
This still passes `--suppress-chown` to the non-chroot command. This should only be added to the `mkosi-chroot` command options and the chroot script options.